package.json

```diff
@@ -23,6 +23,7 @@
   "license": "UNLICENSED",
   "devDependencies": {
     "markdown-proofing": "latest",
-    "shelljs": "^0.7.0"
+    "shelljs": "^0.7.0",
+    "elasticsearch": "^11.0.1"
```

Comment:
Hmm, it's not required to run the site. Perhaps the way to think about it is if `npm install --production` was used, would you want it to be installed or not?